### ISSUE TITLE
Figure02 - Perceptron training

### ISSUE THREAD

**Issue by danielmk** [barisckuru/phase-to-rate#13]
In the final part of Figure02 I want to show the perceptron training and that the learning speed changes systematically with trajectory distance. Therefore, I want to show loss curves for a close, intermediate and distanc trajectory pairings. Next to it I want to show the quantification for non-shuffled with the same learning rate. I am sketching this currently in Miro. Any tips on the best way to get the loss curves with the current perceptron implementation?
